AsyncDisplayKit/ASMapNode.mm

```diff
@@ -190,6 +186,15 @@ - (void)resetSnapshotter
   _snapshotter = [[MKMapSnapshotter alloc] initWithOptions:_options];
 }
 
+- (void)applySnapshotOptions
+{
+  [_mapView setCamera:_options.camera animated:YES];
```

Comment:
Cool!  Pretty awesome that we can use the Options thing to store these kind of parameters even before the view is loaded, even if the only use of the map for that user is as a live map.